libmscore/rangeannotation.cpp

```diff
@@ -0,0 +1,502 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2011 Werner Schweer
```

Comment:
The file created by Ruchit Agrawal in 2016 is copyright 2002-2011 Werner Schweer? ;-)